src/drivers/github.go

```diff
@@ -26,18 +26,23 @@ func (d *githubCodeHostingDriver) CanBeUsed() bool {
 	return d.hostname == "github.com" || strings.Contains(d.hostname, "github")
 }
 
-func (d *githubCodeHostingDriver) CanMergePullRequest(branch, parentBranch string) (bool, error) {
+func (d *githubCodeHostingDriver) CanMergePullRequest(branch, parentBranch string) (bool, string, error) {
 	if d.apiToken == "" {
-		return false, nil
+		return false, "", nil
 	}
 	d.connect()
-	pullRequestNumbers, err := d.getPullRequestNumbers(branch, parentBranch)
+	pullRequests, err := d.getPullRequests(branch, parentBranch)
 	if err != nil {
-		return false, err
+		return false, "", err
 	}
-	return len(pullRequestNumbers) == 1, nil
+	if len(pullRequests) != 1 {
+		return false, "", nil
+	}
+	defaultCommitMessage := fmt.Sprintf("%s (#%d)", *pullRequests[0].Title, *pullRequests[0].Number)
```

Comment:
What about extracting a `defaultCommitMessage(p *github.PullRequest)` method and calling it here? You already have the `pullRequests` data here.

Reply:
done